src/main/java/uk/gov/hmcts/reform/sscs/reference/data/service/PanelCategoryService.java

```diff
@@ -32,6 +32,23 @@ public PanelCategory getPanelCategory(String benefitIssueCode, String specialism
                 .findFirst().orElse(null);
     }
 
+    public PanelCategory getPanelCategoryFromCaseData(SscsCaseData caseData) {
+        String benefitIssueCode = caseData.getBenefitCode() + caseData.getIssueCode();
+        String specialismCount = caseData.getSscsIndustrialInjuriesData().getPanelDoctorSpecialism() != null
+                ? caseData.getSscsIndustrialInjuriesData().getSecondPanelDoctorSpecialism() != null
+                ? "2" : "1" : null;
+        String isFqpm =  isYes(caseData.getIsFqpmRequired()) ? "true" : null;
+        String isMedicalMember =  isYes(caseData.getIsMedicalMemberRequired()) ? "true" : null;
+        PanelCategory panelCategorySelection = new PanelCategory(benefitIssueCode, specialismCount, isFqpm, isMedicalMember);
+        return panelCategories.stream()
+                .filter(panelCategorySelection::equals)
+                .findFirst().orElse(null);
+    }
+
+    public boolean isBenefitIssueCodeValid(String benefitIssueCode) {
```

Comment:
it might be better to pass two arguments and concatenate them in this method, as part of the validation
rather than everywhere it gets called, doing the concatenation